images/prometheus/tests/main.tf

```
@@ -173,3 +174,21 @@ resource "helm_release" "cloudwatch-exporter" {
     value = format("latest@%s", data.oci_string.ref["cloudwatch-exporter"].digest)
   }
 }
+
+resource "helm_release" "pushgateway" {
+  name       = "pushgateway"
+  repository = "https://prometheus-community.github.io/helm-charts"
+  chart      = "prometheus-pushgateway"
+
+  namespace        = "prometheus-pushgateway"
+  create_namespace = true
+
+  set {
+    name  = "image.repository"
+    value = data.oci_string.ref["pushgateway"].registry_repo
+  }
+  set {
+    name  = "image.tag"
+    value = format("latest@%s", data.oci_string.ref["pushgateway"].digest)
```

```suggestion
    value = data.oci_string.ref["pushgateway"].pseudo_tag
```

same result, just fysa